.agents/skills/auto-verify/SKILL.md

```
@@ -44,11 +51,35 @@ npm run typecheck 2>$null; if ($LASTEXITCODE -ne 0) { npx tsc --noEmit }
 npx eslint --max-warnings 0 <changed-files>
 ```
 
-5. Run git hygiene checks:
+5. Check docstring coverage on changed files (whole-file scan, not diff-only):
```

markdownlint-cli2 (0.22.0)

[warning] Ordered list item prefix
Expected: 1; Actual: 5; Style: 1/1/1

(MD029, ol-prefix)